package.json

```diff
@@ -36,10 +36,14 @@
     "gulp-clean": "~0.3.1",
     "gulp-uglify": "~1.1.0",
     "gulp-shell": "~0.3.0",
-    "es6-promise": "~2.0.1"
+    "es6-promise": "~2.0.1",
+    "uglify-js": "~2.4.17"
   },
   "scripts": {
+    "all": "gulp all",
+    "build": "gulp build",
     "test": "gulp test",
+    "test:watch": "gulp test:watch",
```

Comment:
I just verified that the behavior is as I thought. I uninstalled Gulp from my system (`npm uninstall -g gulp`), verified that running `gulp` errored, and then ran `npm run test` and verified that my locally installed Gulp was used instead.